src/react/components/common/cloudFilePicker/cloudFilePicker.tsx

```diff
@@ -63,21 +73,45 @@ export class CloudFilePicker extends React.Component<ICloudFilePickerProps, IClo
         this.onClickConnection = this.onClickConnection.bind(this);
         this.fileList = this.fileList.bind(this);
         this.onClickFile = this.onClickFile.bind(this);
+        this.handleChangeUri= this.handleChangeUri.bind(this);
+        this.handlePasteUri= this.handlePasteUri.bind(this);
 
         this.state = this.getInitialState();
     }
 
     public render() {
         const closeBtn = <button className="close" onClick={this.close}>&times;</button>;
 
-        return(
+        return (
             <Modal isOpen={this.state.isOpen} centered={true}>
                 <ModalHeader toggle={this.close} close={closeBtn}>
                     {this.state.modalHeader}
                 </ModalHeader>
-                <ModalBody>
+                <div className={`shared-string-input-container ${this.state.selectedConnection ? "hide" : ""}`}>
+                    <div className="condensed-list-header bg-darker-2 shared-uri-header">
+                        Shared Project String
+                </div>
+                    {!this.props.connections.length &&
```

Comment:
Need to filter for cloud projects.

Doesn't show when I have only local connections
![image](https://user-images.githubusercontent.com/60453211/87754192-388f1600-c7b9-11ea-8c43-34d33743f83f.png)